dev/com.ibm.websphere.appserver.thirdparty.jsf-2.3/bnd.bnd

```diff
@@ -0,0 +1,91 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
```

Comment:
This should probably just have a copyright of 2019 correct?

Reply:
Good catch - that's right